//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abceg_T_AP.h
//
// Code generation for function 'abceg_T_AP'
//

#ifndef ABCEG_T_AP_H
#define ABCEG_T_AP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abceg_T_AP(double P_init, double V_init, double A_init, double P_wayp,
                double A_wayp, double A_max, double J_max, double J_min,
                double T, creal_T t[21]);

#endif
// End of code generation (abceg_T_AP.h)
